FEATURE_PROCESS.adoc

```diff
@@ -1,5 +1,106 @@
+
+When the signoff is complete, the feature team should notify the release coordinator for the current WildFly release.
+
+=== Done
+
+The release coordinator moves the issue to `Done` status when all work associated with the feature has been merged.
+
+[[feature-team]]
+== The Feature Team
+
+A feature team consists of people in the following roles:
+
+=== Developer
+
+There is a single person in this role: the primary developer of the feature. Other members of the team may do development work, but the developer plays a leading role.
+
+=== Subject Matter Experts
+
+Multiple people can participate in this role. SMEs are knowledgeable in a technology area impacted by the feature.
+
+All feature teams must have at least one person in the Subject Matter Expert role.
+
+SMEs are expected to review all aspects of the feature analysis, implementation, test and documentation.
+
+=== Outside Perspective
+
+Multiple people can participate in this role. The Outside Perspective role is meant to serve two main purposes:
+
+* Attempt to bring a 'user' perspective to the feature team. The end user of the feature is unlikely to have anything close to the expertise of the Developer or a Subject Matter Expert, so things that seem understandable or intuitive to people in those roles may not be so to the end user.
+* Attempt to avoid 'group think' in the development team. The feature may be being developed in accordance with existing plans by a team the does work in the feature's technical area, with the Developer and a Subject Matter Expert part of that team. There's nothing wrong with this, but it's useful to have someone involved who was not part of creating those plans.
+
+Ideally a person in the Outside Perspective role would not be deeply knowledgeable in the feature's general technical area, although at times only people with some level of knowledge will be available. A person in the Outside Perspective role *must not* be someone who is part of a team that works in the feature's technical area.
+
+A person in the Outside Perspective role is not expected to do deep technical review of the feature implementation or tests. (Of course they are welcome to do this if they choose.) They should focus on:
+
+* The Overview, User Stories and Requirements sections of the feature analysis.
+* The feature documentation, including any ancillary material like quickstarts or user guides.
+* Any end-user accessible API associated with the feature.
+* For features at 'community' stability or above, a person in the Outside Perspective role should perform manual verification of the feature (i.e. try it out).
+
+All feature teams for features at 'preview' stability or higher must have at least one person in the Outside Perspective role. Features at `experimental` stability are not required to have anyone in the Outside Perspective role.
+
+[[requirements]]
+== The Requirements
```

Comment:
The text below already exists but I am wondering if we should have some clarification in this doc to say things along the lines of, this process should be followed for:

-  The introduction of a feature at a stability level.
- Further feature development at the same stability level.
- AND for the promotion between stability levels.

In all cases requiring an analysis doc covering the relevant progression.

Reply:
I added this; see "Feature Promotion" after "The Process".